### DISCUSSION TITLE
Scroll to the cursor position

### DISCUSSION THREAD

**Discussion by Proful** [helix-editor/helix#4980] [Q&A] [answered]
Is there any way for me to scroll to the cursor position?

Related functionality:-

- C+d Move half page down
- C+u Move half page up

**Comment by pascalkuthe** [comment]
You can center the view so the (primary) cursor is in the center of the view with `zz`/`zc`, is that what you were looking for?

**Comment by Proful** [comment, marked as answer]
Before
![image](https://user-images.githubusercontent.com/354596/205418657-d6061102-7a04-49f2-b3df-3d133f9bd7fe.png)
After (what I'm looking for)
![image](https://user-images.githubusercontent.com/354596/205418800-533e881c-aded-41ff-9959-b547f07ad953.png)


I'm looking for `zt`
